infra/helper.py

```diff
@@ -750,7 +750,7 @@ def docker_run(run_args, print_output=True, architecture='x86_64'):
 
 def docker_build(build_args):
   """Calls `docker build`."""
-  command = ['docker', 'build']
+  command = ['docker', 'build', '--load']
```

Comment:
Did you find that adding this arg was necessary for fixing your project builds?

I'm not a maintainer here, but I have fixed some builds by upgrading pip and I haven't seen needed to touch anything in `infra/`.

I think removing this should probably make the failing infra related CI checks unnecessary and unblock your PR.

Reply:
Yeah I am wondering the same thing.
Could you please clarify? @micahsnyder

Reply:
I did not intend to commit this change.  I found it is required if you're using docker buildx. I keep this change locally so it works, but didn't mean to add it in my PR.  I will remove it.

Reply:
@DaveLak @DonggeLiu Sorry about that.  Fixed now.